pyproject.toml

```diff
@@ -63,45 +65,46 @@ mkdocstrings = { version = ">=0.23.0" }
 black = "^24.4.2"
 pytest-doctestplus = "^1.2.1"
 
-[tool.poetry.urls]
-"Bug Tracker" = "https://github.com/fulcrumgenomics/prymer/issues"
-
 [build-system]
 requires      = ["poetry-core>=1.6"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.git-cliff.changelog]
-header = ""
-trim = true
-body = """
-{% for group, commits in commits | group_by(attribute="group") %}
-    ## {{ group | upper_first }}
-    {% for commit in commits %}
-        - {{ commit.message | upper_first }} ({{ commit.id | truncate(length=8, end="") }})\
-    {% endfor %}
-{% endfor %}\n
-"""
```

Comment:
Since this is repo/CI configuration, I moved it to the bottom.

So the flow of the `pyproject.toml` goes:

1. Python project config
2. Project config